src/ui.h

```diff
@@ -64,6 +64,9 @@ struct uilist_entry_drawn_info {
     }
 };
 
+struct uilist_entry_drawn_info : entry_drawn_info {
```

Comment:
Same for this inheritance.

Reply:
because the way of drawing for those two kinds of UI thing are not exactly the same, currently the drawn info records are not necessary to to be variant yet, but maybe in future one of them is to be changed.